host/xtest/xtest_20000.c

```diff
@@ -384,6 +386,11 @@ static TEEC_Result obj_corrupt(TEEC_UUID *p_uuid, void *file_id,
 			 ta_dirname, obj_filename);
 
 		get_offs_size(type, block_num, version, &real_offset, NULL);
+		if (real_offset == -1UL) {
```

Comment:
Works for me, I made a new pull request for this here: #179 

Pull whichever version you all like better, then I'll close the other.